concepts/type-conversion/about.md

```diff
@@ -1,10 +1,272 @@
 # About
 
-JavaScript values can be of many types, when performing operations or comparisons, types need to either be _explicitly converted_ to the same type, or _implicitly coerced_ to the same type.
+In JavaScript, values may be of different types. Changing the type of a variable can be done by explicit _type conversion_.
+Besides that, JavaScript also performs _type coercion_ (implicit type conversion) when the context requires it.
 
-Generally, type coercion is avoided because it is averse to unexpected errors. Instead, opt for practices which allow for explicit type conversion.
+## Type Conversion
 
-An overview of these terms can be found on _MDN Web Docs_:
+JavaScript does not have a construct to cast into a (different) type like many other languages but there are built-in helpers that can be used instead.
+Most notably, the global objects `Boolean`, `Number` and `String` can be used as functions to convert a value to the respective type.
 
-- [Type Conversion](https://developer.mozilla.org/en-US/docs/Glossary/Type_Conversion)
-- [Type Coercion](https://developer.mozilla.org/en-US/docs/Glossary/Type_coercion)
+### Converting to a Boolean (Truthy/Falsy Values)
+
+With `Boolean(value)` you can convert any value to a boolean.
+How does that work?
+
+There is a fixed set of values, so called _falsy_ values, that convert to `false`.
+Most importantly, `false`, `0`, empty string, `null`, `undefined` and `NaN` are falsy.
+The [MDN article on "Falsy"][mdn-falsy] shows the complete list.
+
+For all other values, `Boolean` returns `true`.
+These values are called _truthy_.
+
+```javascript
+Boolean(-1);
+// => true
+
+Boolean(0);
+// => false
+
+Boolean(' ');
+// => true
+
+Boolean('');
+// => false
+```
+
+Note that because of the described rules, the following values are truthy in JavaScript although they might count as falsy in other languages.
+
+```javascript
+Boolean('0');
+// => true
+
+Boolean('false');
+// => true
```

Comment:
I'm not much of a Polyglot but I'm curious which languages would consider those strings to be falsy?

In Ruby and Elixir, the concept of falsy is very limited, only `0` and `nil` are falsy.

Reply:
On this page https://javascript.info/type-conversions#boolean-conversion they say `'0'` is falsy in PHP. `'false'` is not really an example from another language, it is rather something that might be unexpected. I run into this last week when I wrote `if (<some environment variable>)`. Then I was confused why the if block was still executed even though the environment variable was set to false. Anyway, I will use the phrasing from the introduction instead so people don't start wondering about which languages are meant here.